pkg/sql/schema_resolver.go

```diff
@@ -643,6 +651,26 @@ func (sr *schemaResolver) FunctionDesc(
 	return g.Function(ctx, descID)
 }
 
+// assertUnsafeBuiltinAccess checks access control for crdb_internal builtin functions.
+func (sr *schemaResolver) assertUnsafeBuiltinAccess(
```

Comment:
A unique check needs to be done for builtins with the `crdb_internal` schema, as they do not go through the normal descriptor checks.